ionc/ion_errors.c

```diff
@@ -25,9 +25,9 @@ const char *ion_error_to_str(iERR err)
         break;
 
     #define ERROR_CODE( name, val ) case name: s = #name; break;
-    #include "ion_error_codes.h"
+    #include "ionc/ion_error_codes.h"
 
-    default:
+        default:
```

Comment:
not sure why we're changing the indent here;  i expect this to line up with `case IERR_NOT_IMPL:` above

Reply:
Fixed